worlds/dkc3/Regions.py

```diff
@@ -921,16 +922,16 @@ def create_region(world: MultiWorld, player: int, active_locations, name: str, l
                 loc_bit    = locationData[1] if (len(locationData) > 1) else 0
                 loc_invert = locationData[2] if (len(locationData) > 2) else False
 
-                location = DKC3Location(player, locationName, loc_id, ret, loc_byte, loc_bit, loc_invert)
+                location = DKC3Location(world.player, locationName, loc_id, ret, loc_byte, loc_bit, loc_invert)
                 ret.locations.append(location)
 
     return ret
 
 
-def connect(world: MultiWorld, player: int, used_names: typing.Dict[str, int], source: str, target: str,
+def connect(world: World, player: int, used_names: typing.Dict[str, int], source: str, target: str,
```

Comment:
since the title of this PR claims to be cleaning up this seems like a good time to either completely remove or at least clean up this function too.